exploration/03_asap_warnings.R

```diff
@@ -0,0 +1,163 @@
+box::use(
+  dplyr,
+  gg = ggplot2,
+  gghdx,
+  lubridate,
+  readr,
+  readxl,
+  tidyr,
+  utils
+)
+
+box::use(
+  ../R/blob_connect
+)
+
+gghdx$gghdx()
+
+#######################
+#### DOWNLOAD DATA ####
+#######################
+
+utils$download.file(
+  url = "https://agricultural-production-hotspots.ec.europa.eu/files/warnings_ts.zip",
+  destfile = zf <- tempfile(fileext = ".zip")
+)
+
+utils$unzip(
+  zipfile = zf,
+  exdir = td <- tempdir()
+)
+
+df_warnings <- readr$read_delim(
+  file = file.path(td, "warnings_ts.csv"),
+  delim = ";"
+)
+
+#############################
+#### BRING IN EMDAT DATA ####
+#############################
+
+df_emdat <- blob_connect$read_blob_file("DF_EMDAT")
+
+################################
+#### FILTER TO FARYAB CROPS ####
+################################
+
+df_faryab <- df_warnings |> 
+  dplyr$filter(
+    asap1_name == "Faryab"
+  ) |> 
+  dplyr$mutate(
+    crop_warning = ifelse(
+      w_crop > 20,
+      0,
+      w_crop %% 10 # just the last digit indicates warning level
+    )
```

Comment:
this `w_crop` values seems pretty weird to me. If its above 20 it is crop warning of 0, otherwise its just the last digit? 

Your probably familiar with it from other work/repos, but can you add an explanation of what w_crop actually means  here.

Reply:
Yeah, added a comment.

Reply:
Super confusing lol. Had to refresh myself.